src/main/java/com/angelbroking/smartapi/orderupdate/OrderUpdateWebsocket.java

```diff
@@ -0,0 +1,163 @@
+package com.angelbroking.smartapi.orderupdate;
+
+import com.angelbroking.smartapi.Routes;
+import com.angelbroking.smartapi.smartstream.models.SmartStreamError;
+import com.angelbroking.smartapi.utils.Utils;
+import com.neovisionaries.ws.client.*;
+import lombok.extern.slf4j.Slf4j;
+
+import java.io.IOException;
+import java.time.LocalDateTime;
+import java.util.List;
+import java.util.Map;
+import java.util.Timer;
+import java.util.TimerTask;
+
+@Slf4j
+public class OrderUpdateWebsocket {
+    private static final int PING_INTERVAL = 10000; // 10 seconds
+    private static final String HEADER = "Authorization";
+    private final Routes routes = new Routes();
+    private final String wsuri = routes.getOrderUpdateUri();
+    private WebSocket ws;
+    private String accessToken;
+    private final OrderUpdateListner orderUpdateListner;
+
+    private Timer pingTimer;
+    private LocalDateTime lastPongReceivedTime = LocalDateTime.now();
+
+    public OrderUpdateWebsocket(String accessToken, OrderUpdateListner orderUpdateListner) {
+        if (accessToken.isEmpty() || Utils.validateInputNullCheck(orderUpdateListner)) {
```

Comment:
use Strings.empty

Reply:
now using StringUtils.isEmpty() which will check both empty and null condition.